tests/test_nat.py

```diff
@@ -1,17 +1,12 @@
 import time
 
 class TestNat(object):
-    def setup_db(self, app_db, asic_db, config_db):
```

Comment:
I would still prefer to keep this one and have wrapper functions in DVS class to return the db handles instead of just accessing them directly (dvs.config_db)

Reply:
Why? I can definitely see some benefits to doing it this way but I'm curious what thoughts you have about it. :)

Reply:
My thoughts weren't written clearly :)
I meant have setup_db(dvs) and DVS class would have methods like get_config_db() etc which setup_db(dvs) would use to initialize self.config_db and use them instead of directly accessing the class DVS vars.

Reply:
Done!